src/query/expression/src/types/timestamp.rs

```diff
@@ -237,6 +237,14 @@ pub fn string_to_timestamp(ts_str: impl AsRef<[u8]>, tz: Tz) -> Option<DateTime<
 }
 
 #[inline]
-pub fn timestamp_to_string(ts: i64, tz: Tz) -> impl Display {
-    ts.to_timestamp(tz).format(TIMESTAMP_FORMAT)
+pub fn timestamp_to_string(ts: i64, tz: Tz, ts_format: &str) -> impl Display + '_ {
+    // 1. If write a wrong format, .format(ts_format) will display the ts_format directly.
```

Comment:
To avoid uninitialized function context:
```suggestion
    debug_assert!(!ts_format.trim().is_empty());
    // 1. If write a wrong format, .format(ts_format) will display the ts_format directly.
```